apollos-church-api/package.json

```diff
@@ -74,6 +75,14 @@
     "get-graphql-schema": "^2.1.2",
     "graphql-schema-linter": "^0.2.0",
     "nodemon": "^2.0.2",
-    "linkemon": "^0.1.1"
+    "linkemon": "^0.1.1",
+    "eslint": "^5.16.0",
+    "eslint-config-airbnb-base": "^14.0.0",
```

Comment:
we still need all of these separate eslint packages? Won't the plugin take care of it?

Reply:
We _could_ but the official guidelines recommend you include them as peerDeps

Reply:
Well - I take that back a bit. We can bundle  `configs` and `parsers` but we need to include plugins as peerDeps. I can fix that in core but we'll have to ship another release first.